README.md

```diff
@@ -1,6 +1,18 @@
-dart_docker
+dart_docker (deprecated)
 ===========
 
+**NOTE**: These images are deprecated.
+
+Please consider using the new Docker Official Images for Dart:
+https://hub.docker.com/_/dart
+
+We plan on discontinuing these at a later stage.
```

Comment:
Could we be more specific and just announce a date so that we can turn down our infrastructure and archive the dart_docker repo? The images won't disappear from google/dart (at least not while they are still being used) but would no longer be updated after that date.

Reply:
Sory, how about we stop maintaining these by end of year 2021?

Reply:
Works for me, though it's a bit long. Perhaps 3 months is enough? They don't serve much purpose anymore and the migration should be fairly easy.

Reply:
Updated